tests.yml

```diff
@@ -0,0 +1,66 @@
+# Run python-versions on {{ taskotron_item }} NVR build
+
+- hosts: localhost
+  remote_user: root
+  vars:
+    testcase: dist.python-versions
+    taskotron_generic_task: true
+    # below are fallback vars for local testing
+    artifacts: ./artifacts
+    taskotron_item: python-gear-0.11.0-1.fc27  # you should really override at least this :)
+    taskotron_supported_binary_arches:
+      - x86_64
+  tasks:
+    - name: Install required packages
+      dnf:
+        name: "{{ item }}"
+        state: latest
+      with_items:
+        - rpm-python
+        - python2-dnf
+        - python2-libarchive-c
+        - python-bugzilla
+        - libtaskotron-core
+        - libtaskotron-fedora
+
+    - name: Make sure taskotron results dir exists
+      # this is for placing results.yml file
+      file:
+        path: "{{ artifacts }}/taskotron"
+        state: directory
+
+    - name: Create work dir
+      tempfile:
+        path: /var/tmp
+        state: directory
+        prefix: task-{{ testcase }}_
+      register: workdir
+
+    - name: Print work dir
+      debug:
+        var: workdir.path
+
+    - name: Compute architectures to download
+      set_fact:
+        download_arches: "{{ taskotron_supported_arches | join(',') }}"
+
+    - name: Print architectures to download
+      debug:
+        var: download_arches
+
+    - block:
+        - name: Download RPMs from Koji
+          shell: >
+            ./download_rpms.py {{ taskotron_item }} {{ workdir.path }}
+            {{ download_arches }}
+            &> {{ artifacts }}/test.log
+
+        - name: Run task
+          shell: >
+            ./python_versions_check.py {{ taskotron_item }} {{ workdir.path }}
```

Comment:
Can we invoke python2 here directly instead of adding a shebang and executable flag? Would that break something?

Reply:
Shouldn't break anything, I believe. I'm simply used to the current approach, that's why I wrote it that way.